azure-pipelines.yml

```
@@ -185,3 +189,45 @@ stages:
             inputs:
               command: "test"
               projects: "its/runner/IntegrationTests.csproj"
+
+  - stage: publish_dogfood
+    displayName: "Publish dogfood extension"
+    condition: and(succeeded(), eq(variables['isMain'], true))
```

NP / disclaimer: not tested

```suggestion
    condition: and(succeeded(), variables['isMain'])
```

---

I'll keep it as is in that it clearly states what we expect (boolean true)
examples shown here
https://learn.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#stage-to-stage-dependencies